test/cql-pytest/cassandra_tests/validation/entities/json_test.py

```diff
@@ -849,11 +849,6 @@ def testToJsonFct(cql, test_keyspace):
             execute(cql, table, "INSERT INTO %s (k, timeval) VALUES (?, fromJson(?))", 0, "\"07:35:07.000111222\"")
             assert_rows(execute(cql, table, "SELECT k, toJson(timeval) FROM %s WHERE k = ?", 0), [0, "\"07:35:07.000111222\""])
 
-            # ================ timestamp ================
-            # Reproduces #7997:
-            execute(cql, table, "INSERT INTO %s (k, timestampval) VALUES (?, ?)", 0, datetime(2014, 1, 1, 0, 0, 0))
-            assert_rows(execute(cql, table, "SELECT k, toJson(timestampval) FROM %s WHERE k = ?", 0), [0, "\"2014-01-01 00:00:00.000Z\""])
-
```

Comment:
Please do not modify tests in the cassandra_tests/ directory - these are tests translated from the Cassandra unit tests, and I don't want to add new tests or delete pieces of tests in these files. Please leave this test unchanged, and the new test you added, testTimestampToJson, please don't put it in the cassandra_tests/ directory but rather in `cql-pytest/test_json.py`. The comment in this file explains its purpose, and among other things it includes : _small tests aiming to reproduce bugs discovered by bigger Cassandra tests._